web-ui/src/main/resources/catalog/components/search/searchoptions/partials/searchoptions.html

```diff
@@ -1,19 +1,77 @@
-<div title="{{'exactMatch-help' | translate}}">
+<div title="{{'exactMatch-help' | translate}}"
+     data-ng-show="optionsConfig.exactMatch">
   <label>
     <input type="checkbox"
            data-ng-model="exactMatch"
            aria-label="{{'exactMatch' | translate}}">
     <span data-translate="">exactMatch</span>
   </label>
 </div>
-<div title="{{'titleOnly-help' | translate}}">
+<div title="{{'titleOnly-help' | translate}}"
+     data-ng-show="optionsConfig.titleOnly">
   <label>
     <input type="checkbox"
            data-ng-model="titleOnly"
            aria-label="{{'titleOnly' | translate}}">
     <span data-translate="">titleOnly</span>
   </label>
 </div>
+<div title="{{'searchAllLanguages-help' | translate}}"
+     data-ng-show="optionsConfig.language"
+     class="row">
+  <div class="col-md-2">
+    <span data-translate="">searchWithLang</span>
+  </div>
+  <div class="col-md-10">
```

Comment:
in case of large strings : 
Maybe add a margin move 
```
<div class="col-md-2">
  <span data-translate="">searchWithLang</span>
</div>
```
=> 
```
<div class="col-md-2">
  <span data-translate="">searchWithLang</span>
</div>
<div title="{{'searchAllLanguages-help' | translate}}"
     data-ng-show="optionsConfig.language"
     class="row">
  <div class="col-md-10">...</div>
</div>
```